File: documentation-examples-kotlin/src/kotlin/dev/typr/foundationskt/docs/mariadb/SetType.kt
package dev.typr.foundationskt.docs.mariadb

import dev.typr.foundationskt.*
import dev.typr.foundationskt.data.*

@Suppress("unused")
class SetType {
    //start
    val setType: MariaType<MariaSet> = MariaTypes.set

    // Create and use sets
    val values: MariaSet = MariaSet.of("read", "write")
    val csv: String = values.toCommaSeparated()
    //stop
}

File: documentation-examples-scala/src/scala/dev/typr/foundationssc/docs/mariadb/SetType.scala
package dev.typr.foundationssc.docs.mariadb
import dev.typr.foundationssc.*
import dev.typr.foundationssc.data.*



@SuppressWarnings(Array("unused"))
object SetType:
  //start
  val setType: MariaType[MariaSet] = MariaTypes.set

  // Create and use sets
  val values: MariaSet = MariaSet.of("read", "write")
  val csv: String = values.toCommaSeparated()
  //stop
